ci/docs-builder.py

```diff
@@ -109,6 +109,14 @@ def main():
     all_content_index_rst = os.sep.join([WORKING_DIR, 'pulp', 'docs', 'all_content_index.rst'])
     copyfile('docs/all_content_index.rst', all_content_index_rst)
 
+    # make the _templates dir
+    layout_dir = os.sep.join([WORKING_DIR, 'pulp', 'docs', '_templates'])
```

Comment:
In order to get the abspath then the os.path.abspath should be called, otherwise the behavior of os.path.join and os.sep.join are the same:

```python
>>> import os
>>> os.path.join('a', 'b')
'a/b'
>>> os.sep.join(['a', 'b'])
'a/b'
```

The difference between both is that the former is a helper module and the later is a plain string:

```python
>>> os.path
<module 'posixpath' from '/home/elyezer/.virtualenvs/pulp-smash/lib64/python3.4/posixpath.py'>
>>> os.sep
'/'
```

With the above said, they will generate the same result and both of them can be used. Personally I like `os.path` more because its meaning is more clear to me. This is not required to be changed though.